app/views/works/_work_form_pseuds.html.erb

```diff
@@ -13,11 +13,11 @@
       <% if has_cocreators %>
         <%= label_tag :remove_me, class: "action" do %>
           <%= check_box_tag :remove_me, "1", false %>
-          <%= ts("Remove Me As Co-Creator") %>
+          <%= ts("Remove me as co-creator") %>
         <% end %>
       <% end %>
   </dd>
 <% end %>
 
-<dt><%= form.label :pseuds_to_add, ts("Add Co-Creators") %></dt>
+<dt><%= form.label :pseuds_to_add, t("works.byline.add_co-creators") %></dt>
```

Comment:
:+1: for updating translations.

Any particular reason some (like the ones five lines above) were left as is?

Seems good otherwise.